<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestFilterDialogBox</class>
 <widget class="QWidget" name="TestFilterDialogBox">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="Filter_Type_groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="2">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>43</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="FilterType_comboBox">
                <item>
                 <property name="text">
                  <string>ФНЧ</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ФВЧ</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Полосовой фильтр</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Режекторный фильтр</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Фильтр Гильберта</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Количество портов:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QSpinBox" name="Port_spinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>2</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="Order_groupBox">
              <property name="title">
               <string>Порядок фильтра</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QSpinBox" name="OrderspinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QGroupBox" name="FreqResponse_groupBox">
              <property name="title">
               <string>Частотные характеристики</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Частота дискретизации, МГц</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_5">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>13</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QComboBox" name="Sample_Freq_comboBox">
                     <item>
                      <property name="text">
                       <string>500</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>1000</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>2000</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>106</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="2">
                <widget class="QGroupBox" name="CutFreq_groupBox">
                 <property name="title">
                  <string>Частота среза</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <widget class="QDoubleSpinBox" name="FreqCut_1_SpinBox">
                    <property name="maximum">
                     <double>3000.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>1.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="Fc1_label">
                    <property name="text">
                     <string>Fc, МГц</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QDoubleSpinBox" name="FreqCut_2_SpinBox">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="readOnly">
                     <bool>false</bool>
                    </property>
                    <property name="maximum">
                     <double>3000.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>5.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="Fc2_label_3">
                    <property name="text">
                     <string>Fc, МГц</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="3">
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>105</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="2">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>43</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
